build/push-images.sh

```diff
@@ -24,10 +24,9 @@ source $ROOT/dev/util.sh
 
 # if parallel utility is installed, the docker push commands will be parallelized
 if command -v parallel &> /dev/null ; then
-  images=$(join_by "," "${ci_images[@]}")
-  ROOT=$ROOT DOCKER_USERNAME=$DOCKER_USERNAME DOCKER_PASSWORD=$DOCKER_PASSWORD SHELL=$(type -p /bin/bash) parallel --halt now,fail=1 --eta -d"," --colsep=" " $ROOT/build/push-image.sh {1} {2} ::: "${images}"
+  ROOT=$ROOT DOCKER_USERNAME=$DOCKER_USERNAME DOCKER_PASSWORD=$DOCKER_PASSWORD SHELL=$(type -p /bin/bash) parallel --will-cite --halt now,fail=1 $ROOT/build/push-image.sh {} ::: "${all_images[@]}"
```

Comment:
nit: no `--eta` tag on this? We do seem to have one for the one above. Thinking this should give us some feedback when the build takes too much time.

Reply:
Yes this was a mistake, thanks!